config/swagger.yaml

```diff
@@ -92,9 +92,7 @@ paths:
             type: object
```

Comment:
I see that `'200': ...` is meant to be a [response object](https://github.com/OAI/OpenAPI-Specification/blob/8a387ac71f7f31aaff5b910f1e0e99ad12ff3ef0/versions/2.0.md#responsesObject). Ok.

Reply:
In particular, see [patterned fields in response object](https://github.com/OAI/OpenAPI-Specification/blob/8a387ac71f7f31aaff5b910f1e0e99ad12ff3ef0/versions/2.0.md#patterned-fields-3).

Reply:
I see a single response is defined [here](https://github.com/OAI/OpenAPI-Specification/blob/8a387ac71f7f31aaff5b910f1e0e99ad12ff3ef0/versions/2.0.md#responseObject): it defines fields `description`, `schema`, `headers`, `examples`.

Here, description and schema are used. Ok.